editorModule.js

```diff
@@ -31,30 +32,272 @@ export default class EditorModule{
         this.CSSEditor.setValue(cssContent);
         this.JSEditor.setValue(scriptContent);
 
-        monaco.languages.register({ id: 'FormItWSM' });
+        // Try to get the autocomplete docs from the remote doxygen host
+        var autocompleteJSON = []
+        try {
+            const autocompleteURL = "https://formit3d.github.io/FormItExamplePlugins/docs/autocomplete.json",
+                response = await fetch(autocompleteURL)
+            autocompleteJSON = await response.json()
+        } catch(e) {
+            console.error("Error fetching autocomplete suggestions!", e);
+        }
+
+        async function ShowAutocompletion(obj) { 
```

Comment:
camel case since not a class name.

Reply:
For ShowAutocompletion? Is it not already in camel case?

Reply:
It is Pascal cased. https://stackoverflow.com/questions/41768733/camel-case-and-pascal-case-mistake

Reply:
I see. I'll get that changed